index.js

```diff
@@ -65,7 +56,7 @@ function searchStream (query, opts) {
         popularity: opts.popularity,
         maintenance: opts.maintenance
       },
-      mapJson (obj) {
+      mapJson: (obj) => {
```

Comment:
Ah!! So THAT's why we had that feature in npm-registry-fetch.

It should be spelled `mapJSON` though.